rest_docs/rest-catalog-open-api.yaml

```diff
@@ -564,12 +648,21 @@ components:
           example: '{ "owner": "Hank Bendickson" }'
           default: '{ }'
 
-    Namespace:
```

Comment:
Moved this near `TableIdentifier`.

Reply:
Do we want to do this in another PR to reduce clutter? I agree they make sense together, as there’s no section for `requests`.

I’d alphabetize actual schema elements and then separate them by a block that does requests. I can open a PR for that if you’d like.

Reply:
Let's leave it as is for now. We'll make minor changes over time, I'm sure.